ipie/addons/ephqmc/trial_wavefunction/eph_trial_base.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+from ipie.trial_wavefunction.wavefunction_base import TrialWavefunctionBase
+
+#NOTE could inherit from TrialWavefunctionBase, but would need to redefine abstract methods.. 
```

Comment:
What do you mean here by redefine the abstract methods?

Reply:
I think inheriting from `TrialWavefunctionBase` would force me to define methods `calc_force_bias` and `half_rotate` for all of my trials objects?
If I inherit from `TrialWavefunctionBase` it throws the following error:
```
TypeError: Can't instantiate abstract class ToyozawaTrial with abstract methods calc_force_bias, half_rotate
```

Reply:
got ya.